### PR TITLE
docs(postgres): warn about deprecated clientMinMessage option

### PR DESCRIPTION
**Pull request by sdepold**
Follow-up to #13720.

It will render warnings when the option is used and point out the deprecation.

### CODE REVIEW

**Comment by sdepold** [comment]
I have added a reference to this ticket here https://github.com/sequelize/sequelize/projects/2#card-73890118

**Review by jesse23 — DISMISSED** [review]

**Comment by jesse23** [comment]
> I have added a reference to this ticket here https://github.com/sequelize/sequelize/projects/2#card-73890118

Thx I will cover this next time in my PR

File: lib/dialects/postgres/connection-manager.js [hunk 1]
```diff
@@ -209,6 +209,11 @@ class ConnectionManager extends AbstractConnectionManager {
       query += 'SET standard_conforming_strings=on;';
     }
 
+    if (this.sequelize.options.clientMinMessages !== undefined) {
+      console.warn('Usage of "options.clientMinMessages" is deprecated and will be removed in v7.');
+      console.warn('Please use "option.dialectOption.clientMinMessages" instead.');
```
Comment (Keimeno) [inline lib/dialects/postgres/connection-manager.js:214]:
```suggestion
      console.warn('Please use "option.dialectOptions.clientMinMessages" instead.');
```

**Review by Keimeno — CHANGES_REQUESTED** [review]

File: lib/dialects/postgres/connection-manager.js (hunk 1 above)
Comment (sdepold) [reply lib/dialects/postgres/connection-manager.js:214]: Oh I copied this from the main branch. There probably is the same typo

**Comment by sdepold** [comment]
> > I have added a reference to this ticket here https://github.com/sequelize/sequelize/projects/2#card-73890118
> 
> Thx I will cover this next time in my PR

Are you saying you will copy my change to your PR or referring to fixing the referenced card on the project?

**Comment by sdepold** [comment]
> > > > I have added a reference to this ticket here https://github.com/sequelize/sequelize/projects/2#card-73890118
> > > 
> > > 
> > > Thx I will cover this next time in my PR
> > 
> > 
> > Are you saying you will copy my change to your PR or referring to fixing the referenced card on the project?
> 
> My PR is merged...I will add this kind of warning next time when I make a change/refactor like this.

Gotcha

**Comment by sdepold** [comment]
retest all

**Review by Keimeno — APPROVED** [review]

**Pull request merged**